(node:1861856) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingLogouts: '/home/student/MITM/logs/logouts',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'nobanner-session-45832',
  containerIp: '10.0.3.21',
  mitmPort: 45832,
  autoAccessFixed: 1
}
2024-11-16 23:18:20.261 - [Info] MITM Version: 2
2024-11-16 23:18:20.263 - [Info] Auto Access Enabled: true
2024-11-16 23:18:20.263 - [Debug] [Init] Auto Access Theshold Achieved: false
2024-11-16 23:18:20.274 - [Info] SSH man-in-the-middle server for 10.0.3.21 listening on 10.0.3.1:45832
2024-11-16 23:38:13.773 - [Debug] [Connection] Attacker connected: 20.81.179.106 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-16 23:38:14.083 - [Debug] [Auth] Attacker 20.81.179.106 trying to authenticate with 'password'
2024-11-16 23:38:14.085 - [Debug] [Auto Access] Attacker: 20.81.179.106, Threshold: 1, Attempts: 1
2024-11-16 23:38:14.085 - [Debug] [Auto Access] Compromising the honeypot
2024-11-16 23:38:14.086 - [Debug] [Auto Access] Adding the following credentials: 'admin:123456Aa'
2024-11-16 23:38:14.321 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2024-11-16 23:38:14.323 - [Debug] [Auth] Valid credentials - Password Authentication
2024-11-16 23:38:14.327 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.21
2024-11-16 23:38:14.411 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2024-11-16 23:38:14.534 - [Debug] [EXEC] Noninteractive mode attacker command: cd ~; chattr -ia .ssh; lockr -ia .ssh
2024-11-16 23:38:14.748 - [Debug] [Connection] Attacker connected: 20.81.179.106 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-16 23:38:14.847 - [Debug] [Auth] Attacker 20.81.179.106 trying to authenticate with 'password'
2024-11-16 23:38:14.848 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2024-11-16 23:38:14.849 - [Debug] [Auth] Attacker: 20.81.179.106 has so far made 1 attempts. Remaining: 5 attempts
2024-11-16 23:38:14.914 - [Debug] [Connection] Attacker closed the connection
2024-11-16 23:38:14.925 - [Debug] [Connection] Attacker connected: 20.81.179.106 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-16 23:38:15.022 - [Debug] [Auth] Attacker 20.81.179.106 trying to authenticate with 'password'
2024-11-16 23:38:15.023 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2024-11-16 23:38:15.024 - [Debug] [Auth] Attacker: 20.81.179.106 has so far made 1 attempts. Remaining: 5 attempts
2024-11-16 23:38:15.083 - [Debug] [Connection] Attacker closed the connection
2024-11-16 23:38:15.099 - [Debug] [Connection] Attacker closed connection
2024-11-16 23:38:15.110 - [Debug] [LXC] Container's OpenSSH server ended connection
2024-11-16 23:38:15.111 - [Debug] [LXC] Container's OpenSSH server closed connection
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
